packages/components/src/resizable-box/index.tsx

```diff
@@ -0,0 +1,128 @@
+/**
+ * WordPress dependencies
+ */
+import { forwardRef } from '@wordpress/element';
+
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+import { Resizable } from 're-resizable';
+import type { ResizableProps } from 're-resizable';
```

Comment:
These can be combined into a single import I think?

```suggestion
import { Resizable, ResizableProps } from 're-resizable';
```

Reply:
That's what I did at first, but ESLint complained:
![image](https://user-images.githubusercontent.com/19592990/136460364-9a2522a0-acfc-44ae-b893-c54d82c6accb.png)

I don't really understand what the issue was, or if there even is an actual issue and not just an ESLint issue, but using `import type` worked. It looks like `eslint-plugin-import` is out-of-date (the version in Gutenberg's main `package.json` is `2.23.4`, but the latest release is [`2.24.2`](https://github.com/import-js/eslint-plugin-import/blob/main/CHANGELOG.md#2242---2021-08-24), so maybe that's the issue?

Reply:
Huh... that's weird. It seems like it's probably an eslint configuration issue with TypeScript 🤔 `import/named` is likely looking at the real JavaScript exports and ignoring the TS type exports (which would be in a separate `.d.ts` file for a compiled package).

Reply:
I figure it's not a blocker for merge, though, right?

Reply:
Nope, not at all!